README.md

```diff
@@ -72,8 +85,58 @@ cf set-env $YOUR_APP_NAME LOGS_CONFIG '[{"type":"tcp","port":"10514","source":"j
+
+To start instrumenting your app:
+
+1. Register the directory containing the DLLs in your app.
+To do so, add the path `C:\Users\vcap\app\datadog\dotNetTracer` in the [`probing`](https://docs.microsoft.com/en-us/dotnet/framework/configure-apps/file-schema/runtime/probing-element) element in your `web.config` file.
+
+**example**:
+```xml
+<runtime>
+    <assemblyBinding xmlns="urn:schemas-microsoft-com:asm.v1">
+        <probing privatePath="C:\Users\vcap\app\datadog\dotNetTracer" />
+    </assemblyBinding>
+</runtime>
+```
+
+2. Setup the appropriate environment variables for your application depending on the OS the app will run on. These can be set directly in the `run.cmd` file.
+
+**example**:
+```
+# Setup the Tracing Environment Variables
+SET COR_ENABLE_PROFILING=1
+SET COR_PROFILER={846F5F1C-F9AE-4B07-969E-05C26BC060D8}
+SET COR_PROFILER_PATH_64=C:\Users\vcap\app\datadog\dotNetTracer\x64\Datadog.Trace.ClrProfiler.Native.dll
+SET COR_PROFILER_PATH_32=C:\Users\vcap\app\datadog\dotNetTracer\x86\Datadog.Trace.ClrProfiler.Native.dll
+SET DD_INTEGRATIONS=\Users\vcap\app\datadog\dotNetTracer\integrations.json
+SET DD_DOTNET_TRACER_HOME=\Users\vcap\app\datadog\dotNetTracer
+SET DD_TRACE_LOG_PATH=\Users\vcap\app\datadog\dotNetTracer\dotnet-profiler.log
+# Redirect logs to the Datadog Agent
+.cloudfoundry\hwc.exe | powershell C:\Users\vcap\app\datadog\scripts\redirect_logs.ps1 2>&1
+```
+
+Alternatively, you can install the [Datadog.Trace.ClrProfiler.Managed Nuget package](https://www.nuget.org/packages/Datadog.Trace.ClrProfiler.Managed) in your app before pushing it to CloudFoundry.
+
+### Linux (.NET Core Apps)
+
+Set the following environment variables in your application:
+```
+# Set your API key to send traces to Datadog
+cf set-env $YOUR_APP_NAME DD_API_KEY $YOUR_DATADOG_API_KEY
+# Setting this environment variable will bundle the .NET Tracing dependencies and enable the
+cf set-env $YOUR_APP_NAME DD_DOTNET_TRACING true
+# environment variables needed to automatically instrument your .NET Application
+cf restage $YOUR_APP_NAME
+```
+
 ### DogStatsD Away!
-You're all set up to use DogStatsD. Import the relevant library and start sending data! To learn more, [check our our documentation](https://docs.datadoghq.com/guides/DogStatsD/). Additionally, we have [a list of DogStatsD libraries](https://docs.datadoghq.com/libraries/) you can check out to find one that's compatible with your application.
+You're all set up to use DogStatsD. Import the relevant library and start sending data! To learn more, [check our our documentation](https://docs.datadoghq.com/guides/DogStatsD/). Additionally, there is [a list of DogStatsD libraries](https://docs.datadoghq.com/libraries/) you can check out to find one that's compatible with your application.
```

Comment:
What are the lib you are referring to here? Dogstatsd libs? 

```suggestion
You're all set up to use DogStatsD. Import the relevant library and start sending data! To learn more, [check the Datadog DogStastD documentation](https://docs.datadoghq.com/guides/DogStatsD/). Additionally, there is [a list of DogStatsD libraries](https://docs.datadoghq.com/libraries/) you can check out to find one that's compatible with your application.
```

Reply:
Yeah, like the C# one if you have a .NET app, the ruby one for ruby apps, etc.